src/pages.rs

```diff
@@ -103,13 +105,23 @@ where
         let Ok(mut info) = ctx.props().pages_info.try_borrow_mut() else {
             return false;
         };
+        let Ok(mut is_page_loading) = ctx.props().is_page_loading.try_borrow_mut() else {
+            return false;
```

Comment:
I think it would be better to return `true`, as this failure doesn't mean the entire process is meaningless.